greet.js

```diff
@@ -0,0 +1,11 @@
+function greet(name) {
```

Comment:
Pretty close here in the module! You are missing an entry point to the app (index.js) which should be located in the root dir next to your package.json.

The logic of handling your cli args should be done there, in addition to requiring in your greet module to run.